netbox/netbox/settings.py

```diff
@@ -233,6 +235,21 @@
 # Storage backend
 #
 
+if STORAGE_BACKEND is not None:
+    if not CONFIG_STORAGES:
+        raise ImproperlyConfigured(
+            "STORAGE_BACKEND and STORAGES are both set, remove the deprecated STORAGE_BACKEND setting."
+        )
+    else:
+        warnings.warn(
+            "STORAGE_BACKEND is deprecated, use the new STORAGES setting instead."
+        )
+
+if STORAGE_CONFIG is not None:
```

Comment:
`STORAGE_CONFIG` defaults to an empty dict if not defined, which triggers the warning. We should set it to None by default.

Reply:
I'm not getting the warning if I don't have it defined.

Reply:
Looks like this was fixed in 4552cea